docs/UserGuide.md

```diff
@@ -275,9 +358,14 @@ Format: `mark INDEX`
   Examples:
```

Comment:
i think the formatting of this can be shifted to the left to aligned with the rest of the UG to maintain consistency